packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/turbos-info.jsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const TurbosInfo = ({ className, currency, has_stop_loss, max_stake = 100, min_stake = 1 }) => {
+    const getTextForStake = (text, amount_of_stake) => (
```

Comment:
You can even move this outside of `TurbosInfo` to line no.7

Reply:
I removed function and make it pure

Reply:
@matin-deriv @kate-deriv it might not update it's translation when it's moved outside and the used is changing the platform language. faced it earlier. please check by adding some existing translated string and switching the language while the component is outside.

Reply:
We check it up with Maryia, everything works as it should be